engine-rest/engine-rest-openapi/src/main/templates/paths/metrics/get.ftl

```diff
@@ -0,0 +1,92 @@
+{
+  <@lib.endpointInfo
+      id = "interval"
+      tag = "Metrics"
+      desc = "Retrieves a list of metrics, aggregated for a given interval." />
+
+  "parameters" : [
+
+    <@lib.parameter
+        name = "name"
+        location = "query"
+        type = "string"
+        desc = "The name of the metric. Supported names:
+                `activity-instance-start`,
+                `activity-instance-end`,
+                `job-acquisition-attempt`,
+                `job-acquired-success`,
+                `job-acquired-failure`,
+                `job-execution-rejected`,
+                `job-successful`,
+                `job-failed`,
+                `job-locked-exclusive`,
+                `executed-decision-elements`,
+                `history-cleanup-removed-process-instances`,
+                `history-cleanup-removed-case-instances`,
+                `history-cleanup-removed-decision-instances`,
+                `history-cleanup-removed-batch-operations`" />
```

Comment:
Shouldn't we define these as `enumValues`? The values are limited by the constants in the `Metrics` class

Reply:
Jep, I agree